modules/nf-core/krona/ktimporttaxonomy/tests/main.nf.test.snap

```diff
@@ -0,0 +1,18 @@
+{
+    "sarscov2 - metagenome - kraken report": {
+        "content": null,
```

Comment:
looks like the assertions are not working ?

Reply:
Updated!